README.md

```diff
@@ -70,33 +70,39 @@ To achieve this MLOS provides:
 ## Python only installation
 
 Some of the examples require only the installation of the mlos Python library. These examples do not use the shared memory infrastructure.
-To use this simplified installation, it's recommended to use the Anaconda python distribution and create a new conda environment:
+
+
+First, download the mlos code using git:
```

Comment:
backticks? if you're trying to reference the python package, else capitalization to be consistent with the project name

Reply:
Sorry, that didn't highlight the way I was expecting.  The comment was in reference the the bare lowercase "mlos" in that sentence.

Reply:
did I do the thing that you wanted? (you can also suggest changes here ;)